precog/utils/timestamp.py

```diff
@@ -66,14 +66,14 @@ def datetime_to_iso8601(timestamp: datetime) -> str:
     """
     Convert datetime to iso 8601 string
     """
-    return timestamp.isoformat()
+    return timestamp.strftime("%Y-%m-%dT%H:%M:%S.%fZ")
```

Comment:
not sure its necessary but to pose the question: is it worth adding a line like 
`timestamp=timestamp.tz_convert('utc')` 
Just before this to ensure everything is UTC before adding the "Z" string?  Or are we confident everythign will be in UTC before this point ?

Reply:
added this check, makes sense